examples/posenet/helperFunc.js

```diff
@@ -0,0 +1,362 @@
+let Architecture = [
```

Comment:
variable name starts with lower case `architecture`

Reply:
Is it constant? Use `const`

Reply:
Add comment about the schema, for example, what does it mean for the number, e.g. 2 in `['conv2d', 2]`?